src/main/java/org/stellar/sdk/SignedPayloadSigner.java

```diff
@@ -0,0 +1,45 @@
+package org.stellar.sdk;
+
+/**
+ * Data model for the <a href="https://github.com/stellar/stellar-protocol/blob/master/core/cap-0040.md#xdr-changes">signed payload signer </a>
+ */
+public class SignedPayloadSigner {
+    private String accountId;
+    private byte[] payload;
+
+    /**
+     * constructor
+     *
+     * @param accountId - the StrKey format of a stellar AccountId
+     * @param payload - the raw payload for a signed payload
+     */
+    public SignedPayloadSigner(String accountId, byte[] payload) {
+        this.accountId = accountId;
+        this.payload = payload;
+    }
+
+    /**
+     * get the StrKey encoded representation of a stellar account id
+     * @return stellar account id in StrKey encoding
+     */
+    public String getEncodedAccountId() {
+        return accountId;
+    }
+
+    /**
+     * get the binary format of a stellar account id, a Ed25519 public key
+     * @return stellar account id in binary format
+     */
+    public byte[] getDecodedAccountId() {
+        return StrKey.decodeStellarAccountId(accountId);
+    }
```

Comment:
maybe `getSigner` and `getRawSigner` are more descriptive names? I always have to think twice about which one of encode/decode corresponds to string/bytes

Reply:
`getDecodedAccountId` doesn't just return a decoded an account ID, it returns the raw payload without a discriminant telling us if it is an ed25519 or not, so this new function isn't easily extensible to other key types if we ever add other key types. Could we make the accessor `default` instead of `public` since it seems like this is for internal use only which will also make it easier for us to change it in the future? Should make naming a little less crucial too since it isn't part of the exported API.

Reply:
@leighmcculloch , yes, can do refactor for this internal use, but am thinking to set this up now for scalability on future, I could change this to use AccountID which has the discriminator on key type, instead of String for account and that removes the extra getter methods down to pure pojo:

```
private org.stellar.sdk.xdr.AccountID accountId

public SignedPayloadSigner(AccountID accountId, byte[] payload) 

...

public AccountID getAccountID()
public byte[] getPayload()

```

Reply:
I refactored to this suggestion [c44b1de9fbe6](https://github.com/stellar/java-stellar-sdk/pull/413/commits/c44b1de9fbe6eb0b5bd98fe22b7771082b185384#diff-c1307eaeda0dc6eea5e80ffcbbf35abc90e6d3c41cdde9607ac6e67f94c4a904)